velox/functions/sparksql/String.h

```diff
@@ -1556,4 +1557,33 @@ struct Empty2NullFunction {
   }
 };
 
+template <typename T>
+struct Base64Function {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Varchar>& result,
+      const arg_type<Varbinary>& input) {
+    result.resize(encoding::Base64::calculateMimeEncodedSize(input.size()));
+    encoding::Base64::encodeMime(input.data(), input.size(), result.data());
+  }
+};
+
+template <typename T>
+struct UnBase64Function {
```

Comment:
It would be nice to focus on one function in each PR, thanks.

Reply:
Sure, will separate this to two PRs.

Reply:
Please also review this: https://github.com/facebookincubator/velox/pull/13238, thanks.